pkgs/shells/dgsh/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit, autoconf, automake, pkgconfig,
+  libtool, check, bison, git, gperf,
+  perl, rsync, texinfo, help2man, gettext
+}:
+
+stdenv.mkDerivation rec {
+  name = "dgsh-${version}";
+  version = "nightly-2017-02-05";
```

Comment:
Versions can't start with a letter. This should be
```
name = "dgsh-unstable";
version = "2017-02-05";
```
See description in the [package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming) section of the Nixpkgs manual.